src/App.test.js

```diff
@@ -11,6 +11,7 @@ it('renders without crashing.', () => {
 
     const { getByText } = Testing.render(
         <App
+            middleEnd={M}
             store={store}
```

Comment:
Do we want to remove `store` here?

Reply:
Yeah, if it is no longer used by `App` then it should be removed here and other places it was previously passed.